app/controllers/api_controller/reports.rb

```diff
@@ -1,12 +1,21 @@
 class ApiController
   module Reports
+    SCHEDULE_ATTR = %w(start_date interval time_zone send_email)
     #
     # Reports Supporting Methods
     #
     def results_query_resource(object)
       object.send("miq_report_results")
     end
 
+    def schedules_query_resource(object)
+      klass = collection_class(:schedules)
+      exp = {}
+      exp["="] = {"field" => "MiqReport.id", "value" => object.id}
+      exp = MiqExpression.new(exp)
```

Comment:
one liner for above: exp = MiqExpression.new("=" => { "field" => ..., "value" => ...})